### DISCUSSION TITLE
quasar upgrade says packages are up to date even though I'm on old version

### DISCUSSION THREAD

**Discussion by chrissyast** [quasarframework/quasar#17606] [General - Components / Directives / etc]
```
    "dependencies": {
        "@quasar/extras": "^1.15.11",
        "@sentry/tracing": "^7.37.2",
        "@sentry/vue": "^7.37.2",
        "@vuelidate/core": "^2.0.0",
        "@vuelidate/validators": "^2.0.0",
        "axios": "^0.21.1",
        "dayjs": "^1.11.7",
        "dayjs-plugin-utc": "^0.1.2",
        "leaflet": "^1.9.3",
        "libphonenumber-js": "^1.10.20",
        "lightgallery": "^2.7.1",
        "pinia": "^2.0.30",
        "powerbi-client": "^2.22.2",
        "powerbi-client-vue-js": "^1.0.3",
        "quasar": "^2.11.6",
        "vscode-icons-js": "^11.6.1",
        "vue": "^3.2.47",
        "vue-gtag": "^2.0.1",
        "vue-i18n": "^9.2.2",
        "vue-matomo": "^4.2.0",
        "vue-router": "^4.1.6",
        "vuedraggable": "^4.1.0"
    },
    "devDependencies": {
        "@quasar/app-vite": "^1.2.0",
        "@quasar/quasar-app-extension-testing": "^2.1.0",
        "@types/jest": "^29.4.0",
        "@types/node": "^18.13.0",
        "@typescript-eslint/eslint-plugin": "^5.52.0",
        "@typescript-eslint/parser": "^5.52.0",
        "autoprefixer": "^10.4.13",
        "eslint": "^8.34.0",
        "eslint-config-prettier": "^8.6.0",
        "eslint-plugin-vue": "^9.9.0",
        "jest": "^29.4.3",
        "prettier": "^2.8.4",
        "supertest": "^6.3.3",
        "ts-jest": "^29.0.5",
        "typescript": "^4.9.5"
    },

```

App vite is version 1.2 (latest version is 1.10.2)
Quasar version is 2.11.6 (latest version is 2.17.1)

When I run `quasar upgrade`:

```
 Global Quasar CLI • Gathering information from the NPM registry (https://registry.npmjs.org/)...
 Global Quasar CLI • Congrats! All Quasar packages are up to date (according to https://registry.npmjs.org/).
```

why is it not updating to the latest version?

**Comment by yusufkandemir** [comment]
`^1.2.0` doesn't mean you have `1.2.0`. If you deleted the lock file and installed dependencies, or you run `npm update`/`yarn update`/`pnpm update`, it will update the dependencies under the hood, but the `package.json` will stay the same. To confirm the actual versions, you can use `quasar info`. But, since you suspect Quasar CLI, you can use your package manager to verify it.

```sh
$ pnpm why @quasar/app-vite
$ yarn why @quasar/app-vite
$ npm list @quasar/app-vite
```

**Reply by TobyMosque** [reply]
if u wanna a fancy ui:
```
$ npx npm-check-updates --interactive
$ yarn upgrade-interactive
$ pnpm update --interactive
```

**Comment by chrissyast** [comment]
I don't "suspect" anything, I'm just confused as to why the upgrade command doesn't upgrade. The command is shown under a heading ["Older v2 to latest v2"](https://quasar.dev/start/upgrade-guide#older-v2-to-latest-v2), but it doesn't seem to be doing that.

**Reply by yusufkandemir** [reply]
I've explained it in the best way I could. I am saying that if you type the commands I gave to you, you will see it's not upgrading anything because **there is nothing to update**.

**Reply by yusufkandemir** [reply]
If it makes you feel better, you can manually "update" versions in `package.json` and see nothing changes when you install the dependencies.

**Reply by chrissyast** [reply]
But why is there nothing to update? Quasar is not on the latest version.

> I've explained it in the best way I could. I am saying that if you type the commands I gave to you, you will see it's not upgrading anything because **there is nothing to update**.

**Reply by yusufkandemir** [reply]
read the highlighted parts and you'll understand that it's actually on the latest version, but you don't believe it, and how you can verify that it's on the latest version so that you would believe
![image](https://github.com/user-attachments/assets/70dc7028-e7c5-41ca-bf85-d6f9109177b1)
